docs/en/sdk-manual/react-native.md

```diff
@@ -0,0 +1,376 @@
+| isCompressEvents            | false    | true          | whether to compress event content by gzip when uploading events                              |
+| isTrackScreenViewEvents     | false    | true          | whether auto record screen view events in app                                                |
+| isTrackUserEngagementEvents | false    | true          | whether auto record user engagement events in app                                            |
+| isTrackAppExceptionEvents   | false    | false         | whether auto track exception events in app                                                   |
+| sendEventsInterval          | false    | 10000         | event sending interval in milliseconds                                                       |
+| sessionTimeoutDuration      | false    | 1800000       | the duration for session timeout in milliseconds                                             |
+| authCookie                  | false    | --            | your auth cookie for AWS application load balancer auth cookie                               |
+| globalAttributes            | false    | --            | the global attributes when initializing the SDK                                              |
+
+#### Configuration update
+
+You can update the default configuration after initializing the SDK, below are the additional configuration options you
+can customize.
+
+```typescript
+import { ClickstreamAnalytics } from '@aws/clickstream-react-native';
+
+ClickstreamAnalytics.updateConfigure({
+  appId: 'your appId',
+  endpoint: 'https://example.com/collect',
+  isLogEvents: true,
+  authCookie: 'your auth cookie',
+  isCompressEvents: true,
+  isTrackScreenViewEvents: false,
+  isTrackUserEngagementEvents: false,
+  isTrackAppExceptionEvents: false,
+});
+```
+
+#### Disable SDK
+
+You can disable the SDK in the scenario you need. After disabling the SDK, the SDK will not handle the logging and
+sending of any events. Of course, you can enable the SDK when you need to continue logging events.
+
+```javascript
+import { ClickstreamAnalytics } from '@aws/clickstream-react-native';
+
+// disable SDK
+ClickstreamAnalytics.disable();
+
+// enable SDK
+ClickstreamAnalytics.enable();
+```
+
+#### Debug events
+
+You can follow the steps below to view the events raw JSON and debug your events.
+
+1. Enable the `isLogEvents` configuration when initializing the SDK
+   ```javascript
+   import { ClickstreamAnalytics } from '@aws/clickstream-react-native';
+   
+   ClickstreamAnalytics.init({
+     appId: 'your appId',
+     endpoint: 'https://example.com/collect',
+     isLogEvents: true,
+   });
+   ```
+
+2. After configuring `isLogEvents:true`, when you record an event, you can see the event raw
+   json in AndroidStudio Logcat or Xcode debug console by filter `EventRecorder`.
+
+## Data format definition
```

Comment:
Is it the same as the web SDK or Android/iOS SDK? If they are same, let's add a link to other SDK section for maintaining duplicated contents.

Reply:
added link